### ISSUE TITLE
Remote roles missing 'auditor' role type

### ISSUE THREAD

**Issue by nahun** [F5Networks/f5-declarative-onboarding#81]
The [RemoteAuthRole](https://clouddocs.f5.com/products/extensions/f5-declarative-onboarding/latest/schema-reference.html#remoteauthrole) class is missing the 'auditor' (80) role in the enum.
https://github.com/F5Networks/f5-declarative-onboarding/blob/37dff4d4541541cc6716c53f23b2c49b50840187/src/schema/1.8.0/auth.schema.json#L169-L188

From TMSH on a 14.1.2 VE:
```
# modify auth remote-role role-info add { test { role ?
Specifies the role that you want to grant to the specified group of remotely authenticated users. The default value is no-access. The available roles and the corresponding number that you use to specify the role are: admin
(0), resource-admin (20), user-manager (40), auditor (80), manager (100), application-editor (300), operator (400), firewall-manager (450), fraud-protection-manager (480), certificate-manager (500), irule-manager (510), guest
(700), web-application-security-administrator (800), web-application-security-editor (810), acceleration-policy-editor (850) and no-access (900).
```

FYI, I am a F5 employee: n.wheeler@f5.com

**Comment by dstokesf5** [comment]
I see that the auditor role is partially missing from the tmsh documentation, so it was likely missed. I have created internal product backlog item AUTOTOOL-902 to track this bug.

**Comment by LeonardosGitHub** [comment]
+1 I'm also looking for the Auditor role to be included.

**Comment by kwinkelman-bw** [comment]
This is impacting our DO configs, would like to see this addressed.

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 1.12.0 release.
